runtime/tests/interpreter/equality_test.go

```diff
@@ -145,10 +145,6 @@ func TestInterpretEqualityOnNumericSuperTypes(t *testing.T) {
 	t.Parallel()
 
 	operations := []ast.Operation{
-		ast.OperationGreater,
-		ast.OperationGreaterEqual,
-		ast.OperationLess,
-		ast.OperationLessEqual,
```

Comment:
Similarly, these tests (i.e: checking the runtime behavior for these operations) are not longer valid, since they don't reach runtime